ros/src/twist_controller/dbw_node.py

```diff
@@ -135,7 +153,7 @@ def upd_dbw_enabled(self, msg):
     def log_init(self, log_path):
         log_handle = open(log_path,'w')
         headers = ','.join(["Time", "Target speed", "Target yaw", "Current speed", "DBW status", "Throttle", "Brake", "Steering"])
-        log_handle.write(headers + '\n')
+        #log_handle.write(headers + '\n')
```

Comment:
Is it possible to keep the headers and adapt the plotting script? It will make it easier if we want to create manual graphs in addition to the plotting script and check manually the values from the file.

Reply:
Yes, I found out that np.loadtext() has an argument to ignore the first x lines in a text file. I've made this change